### ISSUE TITLE
check_tsd references unknown attribute; unusable

### ISSUE THREAD

**Issue by stromnet** [OpenTSDB/opentsdb#760]
tools/check_tsd line 231 tries to read options.bad_percent, which was removed in 010ed96a572f33b35b570cc4c4ce80a5a97b371a:

```
$ python /usr/local/share/opentsdb/tools/check_tsd -H xxxxx -p 4242 -m owfs.reading -t alias=device_cnt -w 34 -c 0 -v
Connected to 2001:xxxxxx:2:4242
send: 'GET /q?start=600s-ago&m=sum:owfs.reading{alias=device_cnt}&ascii&nagios HTTP/1.1\r\nHost: back-1.stromnet.se:4242\r\nAccept-Encoding: identity\r\n\r\n'
reply: 'HTTP/1.1 200 OK\r\n'
header: Content-Type: text/plain
header: Age: 0
header: Cache-Control: max-age=0
header: Content-Length: 4256
owfs.reading 1459792803 34 alias=device_cnt bus=main type=stats_bus ch=None
owfs.reading 1459792863 34 alias=device_cnt bus=main type=stats_bus ch=None
...
owfs.reading 1459796050 34 alias=device_cnt bus=main type=stats_bus ch=None
owfs.reading 1459796110 34 alias=device_cnt bus=main type=stats_bus ch=None

ignored 46/56 data points for being more than 600s old
worse data point value=34 at ts=1459795569
Traceback (most recent call last):
  File "/usr/local/share/opentsdb/tools/check_tsd", line 258, in <module>
    sys.exit(main(sys.argv))
  File "/usr/local/share/opentsdb/tools/check_tsd", line 231, in main
    if options.bad_percent is not None and rv > 0 \
AttributeError: Values instance has no attribute 'bad_percent'
```

check_tsd can thus never output a valid OK value, it crashes before it reaches the OK printer.

Also described in #728.

**Comment by stromnet** [comment]
Fix available in PR #776

**Comment by IDerr** [comment]
@manolama you can close this one, merged here : https://github.com/OpenTSDB/opentsdb/pull/779
